server/dao/listing.js

```diff
@@ -77,6 +79,11 @@ class ListingDao {
     const { rows } = await this.dbPool.query(
       `UPDATE rentr_listing
       SET 
+        is_available = (SELECT
+                CASE WHEN is_available = $11 IS NOT NULL THEN $11 
```

Comment:
Do you think case 11 should go below case 10, just to keep consistent ordering?

Reply:
Yes, I can change the order for consistency.